netbout-web/src/test/java/com/netbout/email/EmAliasTest.java

```diff
@@ -59,19 +65,57 @@ public void sendsEmail() throws Exception {
         final Alias alias = new EmAlias(base.randomAlias(), postman);
         final Bout bout = alias.inbox().bout(alias.inbox().start());
         bout.friends().invite(base.randomAlias().name());
-        bout.messages().post("how are you?");
+        bout.messages().post(EmAliasTest.EMAIL_BODY);
         final ArgumentCaptor<Envelope> captor =
             ArgumentCaptor.forClass(Envelope.class);
         Mockito.verify(postman).send(captor.capture());
         final Message msg = captor.getValue().unwrap();
         MatcherAssert.assertThat(msg.getFrom().length, Matchers.is(1));
+        final InternetAddress from = (InternetAddress) msg.getFrom()[0];
+        MatcherAssert.assertThat(
+            from.getPersonal(),
+            Matchers.is(alias.name())
+        );
+        MatcherAssert.assertThat(
+            from.getAddress(),
+            Matchers.is("no-reply@netbout.com")
+        );
         MatcherAssert.assertThat(msg.getAllRecipients().length, Matchers.is(1));
         MatcherAssert.assertThat(msg.getSubject(), Matchers.startsWith("#1: "));
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
         MimeMultipart.class.cast(msg.getContent()).writeTo(baos);
         MatcherAssert.assertThat(
-            baos.toString(), Matchers.containsString("how are you")
+            baos.toString(), Matchers.containsString(EmAliasTest.EMAIL_BODY)
```

Comment:
@ndbroadbent and ` baos.toString(), Matchers.containsString("how are you")` here